target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -49,6 +49,12 @@ ramips_board_detect() {
 	*"ALL5003")
 		name="all5003"
 		;;
+	*"NetBot")
```

Comment:
Okay, once more: Remove any addition to this file. It isn't required if you finally would use the compatible from the dts files as userspace boardnames.

Reply:
here, we have ramips.sh, it's only working for fixing the boardnames ?, if we want to use the name from dts, we don't have to put anything here?

Reply:
> if we want to use the name from dts, we don't have to put anything here?

correct

Reply:
Ok, I will remove all of them. Thanks for you patience.